thefuck/rules/remove_shell_prompt_literal.py

```diff
@@ -19,4 +19,4 @@ def match(command):
 
 
 def get_new_command(command):
-    return command.script.replace("$", "", 1).strip()
+    return command.script.lstrip(" $")
```

Comment:
Just realized maybe it would make more sense to do `lstrip("$ ")` even though it's the same result

Reply:
I fixed it because it was bothering me